.azure-pipelines.yml

```diff
@@ -269,6 +269,53 @@ jobs:
 #################################################################################
 
 
+- job: 'linux_pypi_build'
+  displayName: 'Linux PyPI build'
+  condition: startsWith(variables['build.sourceBranch'], 'refs/tags/')
+  timeoutInMinutes: 60
+  pool:
+    vmImage: 'ubuntu-latest'
+  steps:
+  - checkout: self
+    submodules: true
+  - bash: |
+      set -eux
+      sudo apt-get install -y python python-pip xvfb xorg
+      pip install twine
+    displayName: 'Bootstrap'
+  - bash: |
+      set -eux
+      python setup.py sdist bdist_wheel
+  - bash: |
+      set -eux
+      pip install --user virtualenv
+      python -m venv venv
+      source venv/bin/activate
+      pip install dist/*.whl[all]
+      xvfb run parsec core gui
```

Comment:
Why do we run this here instead than in the next bash entry ? (seems more logical to keep all the test commands in the same part)

Reply:
I used those test commands here because I wanted to test pip install in a -eux shell. I just forgot to remove the xvfb command...